src/sbt-test/skip-windows/caching/test

```diff
@@ -216,6 +217,7 @@ $ exec chmod 000 src/main/scala/Valid.scala
 > scalafix --check ProcedureSyntax
 -> test:scalafix --check ProcedureSyntax
 > test:scalafix ProcedureSyntax
+> test:scalafix ProcedureSyntax
```

Comment:
a second warm is needed for the cache to converge since there is a rewrite involved